tests/backend/user/test_apiv1_human_find.py

```diff
@@ -8,6 +8,71 @@
 from tests.common import freeze_time, customize_fixtures
 
 
+class NonDeterministicOrderedResults(list):
```

Comment:
A comment would be good to explain the need for this special kind of list

Reply:
done